data-pipeline/src/trace_exporter.rs

```diff
@@ -38,6 +41,13 @@ pub enum TraceExporterOutputFormat {
     V07,
 }
 
+// internal health metrics
```

Comment:
It seems I'e lost that comment in the first review. I expected the following metric names (maybe some customers or TEEs have dashboards using them) with maybe an extra tag to differntiate the source. Is there a particular reason you didn't go that way (I'm fine with it, just want to understand)?
datadog.tracer.queue.dropped_traces
datadog.tracer.queue.dropped_spans
datadog.tracer.queue.enqueued_traces
datadog.tracer.queue.enqueued_spans
datadog.tracer.queue.dequeued_traces
datadog.tracer.queue.dequeued_spans
datadog.tracer.api.requests
datadog.tracer.api.responses
datadog.tracer.api.errors

Reply:
I had reviewed the original list and there is definitely some overlap of the metrics here but a number of these existing .NET metrics aren't common to other tracers and aren't relevant for some of the functionality being measured here (for example, the `queue` metrics don't make a ton of sense since libdatadog doesn't have a "queue" of traces / spans). 

Along that line it felt relevant that libdatadog itself isn't really a "tracer" so instead of referring to the metrics as `tracer` metrics it felt more appropriate to call out that it's coming from `libdatadog`. This also helps segment the health metrics from health metrics that the tracer itself is emitting, since I imagine that there will be metrics  coming from both simultaneously which will help in debugging if an issue exists in the tracer, or within libdatadog. (e.g. the `queue` metrics from .NET, or `datadog.tracer.spans_started` from the go tracer)

Reply:
> aren't common to other tracers
:sadpanda: Betrayal from the .NET tracer :p
Ok thanks for the explanation. Could have been good to have this detailed in the PR description (unless you had mentionned it at standup actually, in case my bad, and 🤐 :p 

If you've started listing discrepencies, should we log somewhere the various metrics emitted per language? 🤔  I mean, if we say we want to improve the OOTB dasbhoard to list tracer emitted metrics, we could be adding metrics from all languages in one graph maybe? Could be part of the other story for the OOTB dasboard improvements.